tools/BUILD

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+
+py_library(
+    name = "archive",
+    srcs = [
+        "__init__.py",
+        "archive.py",
+    ],
+    srcs_version = "PY3",
+    visibility = [
+        "//experimental:__pkg__",
+        "//tests:__pkg__",
```

Comment:
I don't know if this visibility makes sense here? we don't have a `tests` at root?

Reply:
blind copy pasta. Fixed now